packages/gatsby/src/redux/__tests__/run-sift.js

```diff
@@ -183,6 +183,15 @@ if (!process.env.GATSBY_DB_NODES || process.env.GATSBY_DB_NODES === `redux`) {
           mockNodes()[3].id,
         ])
       })
+      it(`return empty array in case of empty nodes`, async () => {
```

Comment:
Looks like we need one last step:
```suggestion
      it(`return empty array in case of empty nodes`, async () => {
        const queryArgs = {}
```

Reply:
Yeap - I am just finishing the fixes locally.

I was about to define query args as:
`const queryArgs = { filter: {}, sort: {} }`

Which one is better?

Reply:
I think we can use your version!